quinn-proto/src/tests/util.rs

```diff
@@ -166,6 +166,15 @@ impl Pair {
         }
     }
 
+    /// Attempts to connect, asserting that the server doesn't accept the
+    /// connection and the client marks the connection as closed.
+    pub(super) fn assert_connection_refused(&mut self) {
```

Comment:
Factoring this into a helper method seems premature with only one caller, given that it's just a few lines.